package.json

```
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json.schemastore.org/package",
   "name": "@sap/csn-interop-specification",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

How about going for beta.1 now?

```suggestion
  "version": "1.0.0-beta.1",
```

---

not yet, let's stick to 0.4.0 for the first npm release